centos-5/Dockerfile

```diff
@@ -9,7 +12,7 @@ COPY macros.dist /etc/rpm/
 # centos5 has an annoying option of sudoers enabled that affects running sudo
 # form an instance without a tty.
 # See https://bugzilla.redhat.com/show_bug.cgi?id=1020147#c9
-COPY sudoers /etc/sudoers
-
-RUN yum install gcc make rpm-build java-1.7.0-openjdk-devel yum-utils sudo epel-release -y && \
-	yum install git -y
+RUN echo -e "\n\n#includedir /etc/sudoers.d" >> /etc/sudoers
+RUN mkdir -p /etc/sudoers.d
```

Comment:
You could collapse this `RUN` directive into the previous `RUN` directive to avoid creating another intermediate layer in the image